rust/yarp-sys/tests/unescape_tests.rs

```diff
@@ -0,0 +1,79 @@
+use std::{ffi::CString, mem::MaybeUninit};
+
+use yarp_sys::{
+    yp_list_empty_p, yp_list_free, yp_list_t, yp_parser_free, yp_parser_init, yp_parser_t,
+    yp_string_free, yp_string_t, yp_string_t_type, yp_unescape_calculate_difference,
+    yp_unescape_manipulate_string, yp_unescape_type_t,
+};
+
+#[test]
+fn unescape_manipulate_string_test() {
```

Comment:
I think this file should be removed because it uses internal APIs that I don't think are of any use if you are using `yp_parse()` (it will already unescape things for you)